src/components/issue-list-item.component.js

```diff
@@ -99,10 +103,10 @@ export const IssueListItem = ({ type, issue, navigation, locale }: Props) => (
         titleStyle={styles.title}
         subtitleStyle={styles.subtitle}
       />
-      <View style={styles.commentsContainer}>
+      <CommentsContainer>
         <Icon name="comment" type="octicon" size={18} color={colors.grey} />
```

Comment:
I'm wondering if we should define a new styled component here (same way as ListItem), and have all this static props in the styled component.
The JSX code would then simply be something like: `<CommentIcon />`

What do you think of this? 🤔